app/models/ncr/work_order.rb

```diff
@@ -203,9 +203,10 @@ def record_changes
       comment_texts = []
       bullet = changed_attributes.length > 1 ? '- ' : ''
       changed_attributes.each do |key, value|
+        former = property_to_s(self.send(key + "_was")) if self.proposal.approved?
```

Comment:
@afeld I believe this is the test where it isn't present. It doesn't explicitly look for the absence of the text, but it is the case where it is not displayed:
https://github.com/18F/C2/blob/master/spec/models/ncr/work_order_spec.rb#L256-L265